src/main/java/com/revature/validation/JsonValidation.java

```diff
@@ -0,0 +1,263 @@
+package com.revature.validation;
+
+import com.revature.domain.Eval;
+import com.revature.domain.EvalComment;
+import com.revature.domain.QuestionComment;
+import com.revature.domain.QuestionEval;
+import com.revature.domain.QuestionPool;
+import com.revature.repositories.BatchRepository;
+import com.revature.repositories.EvalRepository;
+import com.revature.repositories.EvalTypeRepository;
+import com.revature.repositories.PersonRepository;
+import com.revature.repositories.QuestionCommentRepository;
+import com.revature.repositories.QuestionEvalRepository;
+import com.revature.repositories.QuestionRepository;
+import com.revature.repositories.SubjectRepository;
+import javax.validation.ConstraintViolationException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class JsonValidation {
+
+    @Autowired
+    private QuestionRepository questionDao;
+
+    @Autowired
+    private SubjectRepository subjectDao;
+
+    @Autowired
+    private EvalRepository evalDao;
+
+    @Autowired
+    private QuestionEvalRepository questionEvalDao;
+
+    @Autowired
+    private QuestionCommentRepository questionCommentDao;
+
+    @Autowired
+    private EvalTypeRepository evalTypeDao;
+
+    @Autowired
+    private PersonRepository personDao;
+
+    @Autowired
+    private BatchRepository batchDao;
+
+    public void validateQuestionPoolFields(QuestionPool question) {
+
+        validateIdNotSpecified(question.getId());
+
+        if (question.getMaxCommunicationScore() == null) {
+			throw new ConstraintViolationException("Missing required field "
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Missing required field " 14 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1192)